default/customizations/custom_sitreps.txt

```diff
@@ -160,7 +156,7 @@ EffectsGroup
                 Unowned
             ]
             VisibleToEmpire empire = Source.Owner
-            Turn low = [[ETA_NEXT_TURN]] high = [[ETA_NEXT_TURN]]
+            Turn low = (CurrentTurn + LocalCandidate.ETA - 1) high = (CurrentTurn + LocalCandidate.ETA - 1)
```

Comment:
seems unrelated?

Reply:
I'm not sure why I removed the macro and used its content instead, can't remember what trouble it caused to me.
The macro wasn't use in any other place.
I will try putting it back again (in a couple of days) unless you tell me to leave it as is.

Reply:
The macro is maybe slightly clearer that just its expansion, but if you really want to remove it, then I don't really object. But if so, it should be a separate commit at least...